randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1388,8 +1429,11 @@ Extra - room_id: [35]
   > Other to Diffusion Connection Access
       All of the following:
           Morph Ball
-          # Get into tunnel
-          Mid-Air Morph (Intermediate) or Can Bounce in Ball
+          Any of the following:
+              # Get into tunnel
```

Comment:
```suggestion
              # Get into Tunnel
```

Reply:
tunnel isn't a proper noun, no need for capitalization